cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -126,7 +126,7 @@ controller, and serviceaccounts controller.`,
 				os.Exit(1)
 			}
 
-			stopCh := server.SetupSignalHandler(true)
+			stopCh := server.SetupSignalHandlerIgnoringFurtherSignals(true)
```

Comment:
also the cases with `true` you can revert to be just `server.SetupSignalHandler()`.